package.json

```diff
@@ -26,7 +26,8 @@
     "react": "19.0.0",
     "react-native-macos": "^0.78.2",
     "react-native-windows": "^0.78.5",
-    "react-native-mmkv": "^3.3.0"
+    "react-native-mmkv": "^3.3.0",
+    "reactotron-core-contract": "^0.2.5"
```

Comment:
Can this be a devDependency instead of a runtime dependency? (I know it doesn't actually matter when you use Metro, but it would make me feel better about adding a dep 😀 )